.travis.yml

```diff
@@ -0,0 +1,14 @@
+---
+language: node_js
+node_js:
+  - "6"
+  - "8"
+
+notifications:
+  email: false
+
+install:
+  - npm install
+
+script:
+  - npm test
```

Comment:
should we mention this, or let it fallback to default? https://docs.travis-ci.com/user/languages/javascript-with-nodejs/#Default-Build-Script

Reply:
i tend to be deliberate but we could go w/ implicit

Reply:
sure, being explicit works. i wanted to present the option :)